src/Microsoft.AspNet.Server.Kestrel/Http/ListenerSecondary.cs

```diff
@@ -59,7 +59,7 @@ protected ListenerSecondary(ServiceContext serviceContext) : base(serviceContext
 
                                 DispatchPipe.ReadStart(
                                     (_1, _2, _3) => buf,
-                                    (_1, status2, error2, state2) =>
+                                    (_1, status2, errCode, error2, state2) =>
```

Comment:
_1 _2 _3 are all different ignored parameters, because they can't all be called _

There is another bug to track fixing that particular mess